### ISSUE TITLE
Requests to /targets/{target id}/upstream are failing with a strange response message.

### ISSUE THREAD

**Issue by kacbuk001** [Kong/kong#4604]
### Summary

According to official docs it is possible to retrieve Upstream associated to a specific Target, by making a request to "/targets/{target host:port or id}/upstream". Unfortunately my requests are failing with HTTP 404 code and response:
`{"message": "table: 0x4071f4c8table: 0x410b3ec8table: 0x404624d8"}`

I made some digging an according to actual code "/targets" are completely disabled. 
`  -- deactivate endpoints (use /upstream/{upstream}/targets instead)
  ["/targets"] = endpoints.disable,
  ["/targets/:targets"] = endpoints.disable,
  ["/targets/:targets/upstream"] = endpoints.disable,`
You can see full code here: 
https://github.com/Kong/kong/blob/master/kong/api/routes/targets.lua

If the endpoints are disabled then I would expect a 404 code and message like "not found" or "disabled" which would definitely specify what actually happened. These 3 tables looks mysterious and makes me wonder what magical things they are hiding from my eyes.

In conclusion, the endpoints marked as "disable" in the code are returning a strange and unreadable message.

### Steps To Reproduce

1. Create exemplary Upstream
2. Create exemplary Target for that Upstream
3. Make a request to /targets/{target host:port or id}/upstream
or 
1. Make a request to /targets

### Additional Details & Logs

- Docker image: kong:1.1.2-centos
